pkgs/development/python-modules/pyro5/default.nix

```
@@ -0,0 +1,39 @@
+{ buildPythonPackage
+, fetchPypi
+, lib
+, serpent
+, pythonOlder
+, pytest
+}:
+
+let
+  # ignore network related tests, which fail in sandbox
+  ignoredTests = [ "StartNSfunc" "Broadcast" "GetIP" ];
```

and move into the expression please
```suggestion
  disabledTests = [ "StartNSfunc" "Broadcast" "GetIP" ];
```